schema.js

```diff
@@ -0,0 +1,119 @@
+{
+    "_id": "getty_WA0682-001",
+    "source_name": "getty", 
```

Comment:
I don't think we need a link inside each object, and maybe this field shouldn't be inside the `data` at all, move it up into `meta`?